docs/UserGuide.adoc

```diff
@@ -196,26 +208,30 @@ View recipes of dishes served at the restaurant.
 
 Format: `listRecipes`
 
-For each recipe, the list of ingredients and its corresponding amount in units required for the standard serving will be displayed.
+****
+• For each recipe, the list of ingredients and its corresponding amount in units required for the standard serving will be displayed.
 Each recipe will be indexed.
+****
 
 ==== Add Recipe: `addRecipe`
 
 Add recipe of dish served at the restaurant.
 
 Format: `addRecipe r/RECIPE_NAME n/NUMBER_OF_INGREDIENTS 1/INGREDIENT_1&AMOUNT_IN_UNITS 2/INGREDIENT_2&AMOUNT_IN_UNITS ...`
 
-For all ingredient inputs, if ingredient is new, message appears as a prompt for user to `addIngredient` first, before adding recipe.
+****
+• For all ingredient inputs, if ingredient is new, message appears as a prompt for user to `addIngredient` first, before adding recipe.
 
-Each ingredient input should have the format `1/INGREDIENT_1&AMOUNT_IN_UNITS` where ingredient and amount in standardised units are separated by `&`.
+• Each ingredient input should have the format `1/INGREDIENT_1&AMOUNT_IN_UNITS` where ingredient and amount in standardised units are separated by `&`.
 
-`AMOUNT_IN_UNITS` is allowed to take on a decimal and is in the units specified in  listIngredients.
+• `AMOUNT_IN_UNITS` is allowed to take on a decimal and is in the units specified in  listIngredients.
 
-Number of ingredient inputs should correspond to `n/NUMBER_OF_INGREDIENTS`. If it does not, a message will appear to say incorrect number of inputs.
+• Number of ingredient inputs should correspond to `n/NUMBER_OF_INGREDIENTS`. If it does not, a message will appear to say incorrect number of inputs.
```

Comment:
In my opinion, you don't really need to say "a message will appear to say incorrect number of inputs"- "an error message will be shown" is simpler and better language.